frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationTask.tsx

```diff
@@ -122,6 +122,9 @@ const TaskComponent: React.FC<TaskProps> = ({
     ? `${resourcePathFromModel(PipelineRunModel, pipelineRunName, namespace)}/logs/${name}`
     : undefined;
 
+  const enableLogLink =
+    status?.reason !== runStatus.Idle && status?.reason !== runStatus.Pending && path;
```

Comment:
```suggestion
  const enableLogLink: boolean =
    status?.reason !== runStatus.Idle && status?.reason !== runStatus.Pending && !!path;
```
`enableLogLink` is a boolean sounding variable... but your current possible types are `boolean | string | undefined`.

Reply:
Hmm, probably don't want Cancelled either. 

![image](https://user-images.githubusercontent.com/8126518/81100796-22bcc500-8edb-11ea-94fd-402151e730df.png)